src/editor/EditorAdditions.js

```diff
@@ -62,18 +62,22 @@ const EditorAdditions = () => {
 		}
 	}, [ undismissible_prompt, dismiss_text, dismiss_text_alignment ] );
 
-	// Setting editor size as per the popup size.
+	// Setting editor size as per the overlay size or add inline placement.
 	useEffect( () => {
 		const blockEditor = document.querySelector( '.block-editor-block-list__layout' );
 		if ( blockEditor ) {
 			blockEditor.classList.forEach( className => {
 				if ( className.startsWith( 'is-size-' ) ) {
 					blockEditor.classList.remove( className );
+				} else {
+					blockEditor.classList.remove( 'is-placement-inline' );
```

Comment:
This can be done outside of the `blockEditor.classList` loop, without conditions (below line 68 for example). The intention will be more readable this way.